deploy/helm/kubecf/assets/operations/instance_groups/app-autoscaler.yaml

```diff
@@ -0,0 +1,1040 @@
+    bpm:
+      processes:
+      - name: golangapiserver
+        executable: /bin/bash
+        args:
+        - -c
+        - |
+          /var/vcap/jobs/golangapiserver/bin/apiserver_ctl start
+    run:
+      healthcheck:
+        golangapiserver:
+          readiness:
+            exec:
+              command:
+              - curl
+              - --fail
+              - --head
+              - http://127.0.0.1:1080
+- type: replace
+  path: /instance_groups/name=asapi/jobs/name=metricsforwarder/properties/quarks?
+  value:
+    ports:
+    - name: metricsforwarder
+      protocol: TCP
+      internal: 6201
+    - name: mgfdhealthport
+      portocol: TCP
+      internal: 6403
+    bpm:
+      processes:
+      - name: metricsforwarder
+        executable: /bin/bash
+        args:
+        - -c
+        - |
+          /var/vcap/jobs/metricsforwarder/bin/metricsforwarder_ctl start
+    run:
+      healthcheck:
+        metricsforwarder:
+          readiness:
+            exec:
+              command:
+              - curl
+              - --fail
+              - --head
+              - http://127.0.0.1:6403
+- type: replace
+  path: /instance_groups/name=postgres_autoscaler/jobs/name=postgres/properties/quarks?
+  value:
+    ports:
+    - name: postgres
+      protocol: TCP
+      internal: *postgresPort
+    run:
+      healthcheck:
+        postgres:
+          readiness:
+            exec:
+              command:
+              - /var/vcap/packages/postgres-11.5/bin/pg_isready
+    pre_render_scripts:
+      jobs:
+      - |
+        sed -i "s/sysctl/#sysctl/g" /var/vcap/all-releases/jobs-src/postgres/postgres/templates/pre-start.sh.erb
+    bpm:
+      processes:
+      - name: postgres
+        limits:
+          open_files: 1048576
+        persistent_disk: true
+        executable: /usr/bin/su
+        args:
+        - "-"
+        - vcap
+        - -c
+        - |
+          set -o errexit -o nounset
+
+          wait_for_file() {
+            local file_path="$1"
+            local timeout="${2:-30}"
+            until [[ -f "${file_path}" ]] || [[ "$timeout" == "0" ]]; do sleep 1; timeout=$(expr $timeout - 1); done
+            if [[ "${timeout}" == 0 ]]; then return 1; fi
+            return 0
+          }
+
+          source /var/vcap/jobs/postgres/bin/pgconfig.sh
+
+          /var/vcap/jobs/postgres/bin/postgres_ctl start
+          wait_for_file "${PIDFILE}" || {
+            echo "${PIDFILE} did not get created"
+            exit 1
+          }
+          trap '/var/vcap/jobs/postgres/bin/postgres_ctl stop' EXIT
+
+          /var/vcap/jobs/postgres/bin/pg_janitor_ctl start &
+
+          tail \
+            --pid "$(cat "${PIDFILE}")" \
+            --follow \
+            "${LOG_DIR}/startup.log" \
+            "${LOG_DIR}/pg_janitor_ctl.log" \
+            "${LOG_DIR}/pg_janitor_ctl.err.log"
+
+# TODO: set these skip_ssl_validation to false if the cf API (api.((system_domain))) public cert is provided by the user.
```

Comment:
I assume this is because those jobs can't take extra certs (only the system ones)?
How will that work if the user provides a non-public cert (as in, a custom cert that chains up to their company-local CA)?

Reply:
If the cert is trust and the `skip_ssl_validation` can be set to `false`. But if the cert is not trust(if access the url with the cert from browser, it reports "Your connection is not secure"), the `skip_ssl_validation` value should be `true`.

Reply:
Right, those would be the system certs (VeriSign, Let's Encrypt, CNNIC, Microsoft, …)
I'm thinking about cases of actual deployment: for example, SUSE has an internal CA, which is used for some internal services.  That CA does not sign anything public (and is not in the standard list of public CAs), but is deployed to any internal systems to be trusted. How would that be propagated correctly in this case?